cli/src/cli.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+import logging
+import click
+from rich.logging import RichHandler
+from .authorization import commands as authorization
+from .phpmyadmin import commands as phpmyadmin
+from .ssh import commands as ssh
+
+FORMAT = '%(message)s'
+# NOTSET
+logging.basicConfig(level='INFO', format=FORMAT,
+                    datefmt='[%X]', handlers=[RichHandler()])
+logging.getLogger('paramiko').setLevel(logging.WARNING)
+LOG = logging.getLogger(__name__)
+
+
+@click.group()
+def entry_point():
+    """Manage Meirim"""
+    pass
+
+
+entry_point.add_command(authorization.authorization)
+entry_point.add_command(phpmyadmin.phpmyadmin)
+entry_point.add_command(ssh.ssh)
```

Comment:
to my understanding, the authorization and the ssh modules are suffice here.

Reply:
I strongly disagree. The tool should be a one-stop tool for any development purpose.
I do agree that the hierarchy should be minimal, thus phpMyAdmin is not appropriate here, rather it should be nested under something else. However, this is a work in progress, and I didn't want to create a deeper hierarchy while I got only one item at hand, so I put it in the top level. We will definitely want to move it later on. There will be many more such tools in the future.